src/main/java/io/pinecone/configs/PineconeConfig.java

```diff
@@ -176,7 +176,7 @@ public String getUserAgentGrpc() {
     }
 
     private String buildUserAgent(String clientId) {
-        String userAgent = String.format("lang=java; %s=%s", clientId, "v0.8.0");
+        String userAgent = String.format("lang=java; %s=%s", clientId, "v1.0.0");
```

Comment:
In the future PR, I'm going to automate this to extract version from build.gradle or some other file so we only have to update this version at one place